apps/web/src/server/app/inspector/inspector.controller.js

```diff
@@ -0,0 +1,24 @@
+import { findAppealById } from './inspector.service.js';
+
+/** @typedef {import('@pins/inspector').Appeal} Appeal */
+
+/**
+ * @typedef {Object} AppealParams
+ * @property {string} appealId – Unique identifier for the appeal.
+ */
+
+/**
+ * @typedef {Object} ViewAppealDetailsRenderOptions
+ * @property {Appeal} appeal – The appeal entity.
+ */
+
+/**
+ * View the appeal details as an inspector.
+ *
+ * @type {import('@pins/express').QueryHandler<AppealParams, ViewAppealDetailsRenderOptions>}
+ */
+export async function viewAppealDetails({ params }, response) {
+	const appeal = await findAppealById(params.appealId);
+
+	response.render('inspector/appeal-details', { appeal });
```

Comment:
I think we should add the route path and the views in the config file like we did in the validation, just in case we need to use them in multiple parts of our app.

Reply:
So this was an intentional departure from the routes config way of doing things, both for path and view:

Regarding views, what is actually gained by maintaining an external map of paths to views? I'd argue it makes more sense that the controller function has ownership of  is the only place its rendered view should be declared. Furthermore, template names (and indeed paths) are not easily read without mentally constructing the view path by swapping between files.

Regarding path, importing the route path only really works if it doesn't need to be interpolated in any way. For example, imagine if you have in your config:

`path: 'appeals/:appealId/some-action'`

It is not really possible to use the config approach now, because the different parts of the url require a dynamic id to be inserted in the middle, so some kind of string interpolator would be needed. Alternatively, you could maintain one path as two different entries in the config and start combining them to form urls, but that too doesn't make sense: I saw we were already doing things like `${config.routes.somePath}/${appealId}` in the controllers. That already seems like an anti-pattern because it assumes you already know what the construction of the path without referring to the config at all. Likewise, we are manually prepending `/validation` as a string to the routes in the controller, which is also contradictory to the idea of the config pattern.

Overall, I feel like like the config approach has too much overhead for no real benefit. My preference would be that the <domain>.route.js is the source of truth of routes, and their usage is akin to typing out the name of a constant. If you look at the **validation.route.js** I raised, I find that the routes I added as complete strings give a much clearer overview of the route layout of the domain than the config approach.

Anyway, I'll change it if you're adamant! Let me know!